index.bs

```diff
@@ -0,0 +1,252 @@
+<pre class='metadata'>
+Title: Autoplay Policy Detection
+Shortname: autoplay-detection
+Level: 1
+Status: w3c/ED
+Group: mediawg
+URL: http://example.com/url-this-spec-will-live-at
+Editor: Alastor Wu, Mozilla https://www.mozilla.org, alwu@mozilla.com
+Editor: Paul Adenot, Mozilla https://www.mozilla.org, padenot@mozilla.com
+Abstract: This specification provides web developers the ability to detect if automatically starting the playback of a media file is allowed in different situations.
+Markup Shorthands: markdown on
+</pre>
+
+<style>
+@media (prefers-color-scheme: light) {
+  :root {
+    --div-info-fg-color: #178217;
+  }
+}
+@media (prefers-color-scheme: dark) {
+  :root {
+    --div-info-fg-color: springgreen;
+  }
+}
+div.enum-description > table {
+	border-collapse: collapse;
+	border-top: 2px solid #707070;
+	border-bottom: 2px solid #707070;
+	width: 100%;
+	margin: 2em 0;
+}
+div.enum-description > table > tbody > tr > th,
+div.enum-description > table > tbody > tr > td {
+	padding: 0.2em 0.6em;
+	min-width: 150px;
+	border-top: 1px solid #ddd
+}
+div.enum-description > table > thead > tr > th {
+	line-height: 2em;
+	font-weight: 600;
+	color: var(--div-info-fg-color);
+	border-bottom: 1px solid #707070;
+}
+</style>
+
+<h2 id="introduction">Introduction</h2>
+  Most user-agents have their own mechanisms to block autoplay media, and those mechanisms are implementation-specific. Web developers need to have a way to detect if the autoplay media is allowed or not in order to make actions, such like selecting alternate content or improving the user experience while media is not allowed to autoplay. For instance, if a user-agent only blocks audible autoplay, then web developers can replace audible media with inaudible media to keep media playing, instead of showing a blocked media which looks like a still image to users. If the user-agent does not allow any autoplay media, then web developers could stop loading media resource and related tasks to save the bandwidth and CPU usage for users.
```

Comment:
I'd recommend wrapping lines since this is pretty hard to review otherwise.

Reply:
Done, thank you!